pkg/agent/utils.go

```diff
@@ -25,6 +26,37 @@ import (
 	"github.com/pkg/errors"
 )
 
+var TranslatedKubeletConfigFlags map[string]bool = map[string]bool{
```

Comment:
nit: KubeletConfigFlagsInFile?

Reply:
I'll keep this one since I'd like to call out these are translated from flags